system-parachains/encointer/Cargo.toml

```diff
@@ -0,0 +1,222 @@
+[package]
+name = 'encointer-runtime'
+# major.minor revision must match collator node
+# patch revision must match runtime spec_version
+version = '1.5.16'
```

Comment:
it is unclear to me if we should change to the spec versioning scheme of other system chains. happy to, but need to investigate implications if encointer doesn't update at the same rhythm